recipes/eigen/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release ..
+make install
+make basicstuff
+ctest -R basicstuff*
```

Comment:
Same basic test suite as was used on Windows for the same reasons. Thanks @patricksnape for adding this.